Gems/Atom/RPI/Code/Source/RPI.Public/ViewGroup.cpp

```diff
@@ -0,0 +1,258 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <Atom/RPI.Public/RPISystemInterface.h>
+#include <Atom/RPI.Public/ViewGroup.h>
+#include <AzCore/Debug/Trace.h>
+
+namespace AZ::RPI
+{
+    ViewGroup::ViewGroup()
+    {
+        m_cameraViews.resize(MaxViewTypes);
+    }
+
+    ViewGroup::~ViewGroup()
+    {
+        m_cameraViews.clear();
+    }
+
+    void ViewGroup::Init(Descriptor desc)
+    {
+        m_desc = desc;
+
+        if (auto rpiSystemInterface = RPISystemInterface::Get())
+        {
+            m_xrSystem = rpiSystemInterface->GetXRSystem();
+            if (m_xrSystem)
+            {
+                m_numSterescopicViews = m_xrSystem->GetNumViews();
+                AZ_Assert(m_numSterescopicViews <= XRMaxNumViews, "Atom only supports two XR views");
+            }
+        }
+
+        for (int i = 0; i < MaxViewTypes; i++)
+        {
+            m_cameraViews[i].m_onProjectionMatrixChangedHandler = MatrixChangedEvent::Handler(
+                [=](const AZ::Matrix4x4& matrix)
```

Comment:
is it necessary to use copy scope? Could it use another one with less baggage? Passing `this` maybe?